<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-8DUO6O13" failures="3" tests="13" name="LoginAndSearchTest_Chrome" time="131.447" errors="0" timestamp="2025-09-15T10:50:49 IST">
  <testcase classname="tests.LoginTest" name="verifyLoginTest" time="3.123"/>
  <testcase classname="tests.LoginTest" name="invalidLoginTest" time="5.694"/>
  <testcase classname="tests.LoginTest" name="validLoginTest" time="15.225"/>
  <testcase classname="tests.SearchTest" name="testBlankSearch" time="2.367"/>
  <testcase classname="tests.SearchTest" name="testSearchProduct" time="4.101"/>
  <testcase classname="tests.SearchTest" name="testApplyFilters" time="1.428"/>
  <testcase classname="tests.SearchTest" name="testSortLowToHigh" time="1.309"/>
  <testcase classname="tests.SearchTest" name="testViewProductDetails" time="8.198"/>
  <testcase classname="tests.CartTest" name="cartAddTest" time="32.955"/>
  <testcase classname="tests.CartTest" name="modifyQuantity" time="10.606"/>
  <testcase classname="tests.CartTest" name="deleteFromCart" time="10.076">
    <failure type="java.lang.AssertionError" message="❌ Product removal message not shown! expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: ❌ Product removal message not shown! expected [true] but found [false]
at tests.CartTest.deleteFromCart(CartTest.java:40)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
... Removed 18 stack frames]]>
    </failure>
  </testcase> <!-- deleteFromCart -->
  <testcase classname="tests.CartTest" name="buyProducts" time="20.275">
    <failure type="java.lang.AssertionError" message="❌ Address does not match expected! expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: ❌ Address does not match expected! expected [true] but found [false]
at tests.CartTest.buyProducts(CartTest.java:50)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
... Removed 18 stack frames]]>
    </failure>
  </testcase> <!-- buyProducts -->
  <testcase classname="tests.CartTest" name="placeOrder" time="10.182">
    <failure type="java.lang.AssertionError" message="❌ Cannot place order! expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: ❌ Cannot place order! expected [true] but found [false]
at tests.CartTest.placeOrder(CartTest.java:57)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
... Removed 18 stack frames]]>
    </failure>
  </testcase> <!-- placeOrder -->
</testsuite> <!-- LoginAndSearchTest_Chrome -->
